modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/spring/SimpleTypeBeanParamDefinitionCreator.java

```diff
@@ -43,7 +44,8 @@ protected boolean doHandleRequest(CreateParamBeanDefinitionRequest createBeanDef
 
   @Override
   protected boolean isExpressionValue(CreateParamBeanDefinitionRequest request) {
-    return request.getParam().getValue().isLeft();
+    return request.getParam().getValue().isLeft()
+        || isContent(request.getParam().getModel());
   }
```

Comment:
There is something that I don't understand here. The method name is `isExpressionValue` and we are changing it so that static values are also considered as expressions if the parameter is a content parameter?
Maybe I'm getting it wrong, but from the test it looks like it.

Are we sure this is the right place to add the "if"? or at least maybe we need a good comment here.